lib/rules/require-default-props.js

```diff
@@ -55,14 +62,7 @@ module.exports = {
      * @return {void}
      */
     function reportPropTypesWithoutDefault(propTypes, defaultProps) {
-      // If this defaultProps is "unresolved", then we should ignore this component and not report
-      // any errors for it, to avoid false-positives with e.g. external defaultProps declarations or spread operators.
-      if (defaultProps === 'unresolved') {
-        return;
-      }
-
-      Object.keys(propTypes).forEach((propName) => {
-        const prop = propTypes[propName];
+      Object.entries(propTypes).forEach(([propName, prop]) => {
```

Comment:
Object.entries isn’t available in every node we support; please use the `object.entries` package for that.